src/Git/Pack/Delta.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
Not obviously seeing what this is for, enlightenment requested ;-p

Reply:
You can't specify `MonadError GitError m =>` contexts without it. I'm not sure as to the exact rules as to why.